docs/how-to/configure.md

```diff
@@ -106,15 +115,13 @@ This section lets you upload your public SSH key to access the PMM Server via SS
 
 Enter your **public key** in the *SSH Key* field and click *Apply SSH Key*.
 
-## Alertmanager integration
-
-The Prometheus Alertmanager manages alerts from Prometheus, deduplicating, grouping, and routing them to the appropriate receiver or display component.
+## VictoriaMetrics alerts
```

Comment:
Pls recheck this on the Latest build, I hope it's NOT changed on the UI.
It's still Alertmanager integration.